source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -1348,8 +1348,19 @@ def _tableHeaderTextHelper(self, axis):
 			# Each header must be fetched from the headers array once and only once,
 			# as it gets released when it gets garbage collected.
 			for i in range(nHeaders):
+				header = headers[i]
+				headerIA2Ptr = header.QueryInterface(IA2.IAccessible2)

```

Comment:
Please catch possible COMError around this line, and if so, log a debugWarning and continue.